15_setup_structure (#26)
nestauk/nesta_ds_utils: README.md

# nesta_ds_utils

> This repo contains a python package with basic data science utility functions to assist the DAP team. 

Reviewer: ```suggestion
A Python package with basic utility functions for data science projects at Nesta.
```

> ### This package is currently under development

Reviewer: Doesn't have to be this PR, but could be worth adding some contributor guidelines and some info on requesting/suggesting new features.

Author: Yes, that's a good point. I will open a new issue for that that can be addressed in a different PR, at least on the meantime we  have the basic structure there.
